environments/production/inventory.ini.j2

```diff
@@ -174,7 +174,7 @@ hqcelery4.internal-va.commcarehq.org
 {{ celery0-production }} hostname=celery0-production ec2=yes
 
 [celery1]
-{{ celery1-production }} hostname=celery1-production ec2=yes #datadog_integrations=celery
+{{ celery1-production }} hostname=celery1-production ec2=yes
```

Comment:
Did you intend to enable this rather than deleting it?

Reply:
Embedded the reason in the commit message:
> It turns own that datadog_integrations=celery doesn't do anything,
and that it always picks the first celery machine.

So after trial and error I realized having this variable set here was useless and that I needed to move celery1 to the top of the list instead.